### ISSUE TITLE
LUA Errors on logging in

### ISSUE THREAD

**Issue by NoNick96** [SLOKnightfall/BetterWardrobe#320]
```
3x BetterWardrobe/Modules/Tooltips.lua:9: Cannot find a library instance of "LibAppropriateItems-1.0".
[string "=[C]"]: in function `error'
[string "@Auctionator/Libs/LibStub/LibStub.lua"]:23: in function `LibStub'
[string "@BetterWardrobe/Modules/Tooltips.lua"]:9: in main chunk

Locals:
(*temporary) = "Cannot find a library instance of "LibAppropriateItems-1.0"."
```

```
3x Interface\FrameXML\Bindings.xml:25 Couldn't open BetterWardrobe/Libs/LibAppropriateitems-1.0/lib.xml
```

```
3x BetterWardrobe/Modules/Tooltips.lua:1 (null)
```

```
3x Interface\FrameXML\Bindings.xml:1 Interface\FrameXML\Bindings.xml:25 Couldn't open BetterWardrobe/Libs/LibAppropriateitems-1.0/lib.xml
```

```
3x Interface\FrameXML\Bindings.xml:1 BetterWardrobe/Modules/Tooltips.lua:1 (null)
```

Addon version 4.4.7

**Comment by NoNick96** [comment]
I've disabled all addons besides BetterWardrobe and Bugsack the same thing happens

**Comment by NoNick96** [comment]
Fixed in 4.4.7.1
